File: code/classes/declaration.ts
class Person {
    private name: string;
    private age: number;
    
    constructor(name: string, age: number) {
        this.name = name;
        this.age = age;
    }
    
    getName(): string {
        return this.name;
    }
    
    getAge(): number {
        return this.age;
    }
}

let person: Person = new Person("John", 30);

File: code/classes/declaration.java
// Class declaration in Java
public class Person {
    private String name;
    private int age;
    
    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }
    
    public String getName() {
        return name;
    }
    
    public int getAge() {
        return age;
    }
    
    public String greet() {
        return "Hello, I'm " + name + " and I'm " + age + " years old";
    }
    
    public static void main(String[] args) {
        Person person = new Person("John", 30);
        System.out.println(person.greet());
    }
}